pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Producer.java

```diff
@@ -257,6 +285,9 @@ public ServerCnx getCnx() {
         private String originalProducerName;
         private long originalSequenceId;
 
+        private long lowestSequenceId;
```

Comment:
What does it mean the lowest sequence id of a producer?

Reply:
It is in the inner class named MessagePublishContext of Producer

Reply:
Ok, though as in other comment, there's already a `sequenceId` field. Is that now ignored? If yes, then it should be removed. 

Also, as you can see, there's an `originalSequenceId` field. This is used in the context of geo-replication and it would have to be accounted for as well.